doc/source/examples/dem/granuheap/granuheap.rst

```diff
@@ -0,0 +1,396 @@
+  set frequency           = 100000
+  end
+  set particle particle contact force method = hertz_JKR
+  set rolling resistance torque method       = constant_resistance
+  set particle wall contact force method     = JKR
+  set integration method                     = velocity_verlet
+  end
+
+  subsection restart
+  set checkpoint = true
+  set frequency  = 10000
+  set restart    = false
+  set filename   = dem
+  end
+
+Particle Insertion
+~~~~~~~~~~~~~~~~~~~~
+
+An insertion box is defined inside the cylinder. 10 000 particles are inserted every 9000 iterations. It is important to note the size of the insertion box to ensure it is completely inside our cylinder which is smaller than our domain. Otherwise, particles will be lost during the insertion stage given that the domain's bottom is an outlet.
+
+.. code-block:: text
+
+  subsection insertion info
+  set insertion method                               = volume
+  set inserted number of particles at each time step = 10000
+  set insertion frequency                            = 9000
+  set insertion box points coordinates               = 0.015, -0.00325, -0.00325: 0.035, 0.00325, 0.00325
+  set insertion distance threshold                   = 1.2
+  set insertion maximum offset                       = 0.05
+  set insertion prn seed                             = 19
+  set insertion direction sequence                   =  1, 2, 0
+  end
+
+
+Solid Object
+~~~~~~~~~~~~
+
+We set the cylinder's translational velocity to :math:`0.05 m/s` after the particles were loaded, thus when the simulation time is higher than 1.6s. 
+
+.. code-block:: text
+
+  subsection solid objects
+  subsection solid surfaces
+  set number of solids = 2
+  subsection solid object 0
+  subsection mesh
+  set type                   = gmsh
+  set file name              = cylinder.msh
+  set simplex                = true
+  end
+  subsection translational velocity
+  set Function expression = if (t>1.6, 0.05, 0) ; 0 ; 0
+  end
+  end
+  subsection solid object 1
+  subsection mesh
+  set type                   = gmsh
+  set file name              = support.msh
+  set simplex                = true
+  end
+  end
+  end
+  end
+
+Simulation Control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The process duration lasts for 2.2 seconds. We output the simulation results in every 1000 iterations.
+
+.. code-block:: text
+
+  subsection simulation control
+  set time step         = 7.54e-6
+  set time end          = 2.2
+  set log frequency     = 1000
+  set output frequency  = 1000
+  set output path       = ./output/
+  set output name       = granuheap
+  set output boundaries = true
+  end
+    
+-----------------------
+Running the Simulation
+-----------------------
+
+Running one case
+~~~~~~~~~~~~~~~~
+
+A simulation with one set of values for the ``rolling friction`` and the ``surface energy`` can be launched using the following command:
+
+.. code-block:: text
+  :class: copy-button
+
+  mpirun -np 8 lethe-particles granuheap.prm
```

Comment:
```suggestion
  mpirun -np 12 lethe-particles granuheap.prm
```

Reply:
Cléo, did you use 8 or 12 cores to run the simulation? I know your desktop have 12 cores, but you are giving the time it takes to run the full simulation (5 hours) right after.

Reply:
In the end it does not really matter, as long as the time is reported on a given number of cores